test/e2e/cht-conf-utils.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const { exec } = require('child_process');
+const fs = require('fs');
+const fse = require('fs-extra');
+
+const log = require('../../src/lib/log');
+const { getProjectUrl } = require('./cht-docker-utils');
+
+const getProjectDirectory = (projectName) => path.resolve(__dirname, `../../build/${projectName}`);
+
+const runChtConf = (projectName, command) => new Promise((resolve, reject) => {
+  getProjectUrl(projectName).then(url => {
+    const projectDirectory = getProjectDirectory(projectName);
+    const cliPath = path.join(__dirname, '../../src/bin/index.js');
+    exec(`node ${cliPath} --url=${url} ${command}`, { cwd: projectDirectory }, (error, stdout, stderr) => {
+      if (!error) {
+        return resolve(stdout);
+      }
+
+      log.error(stderr);
+      reject(new Error(stdout.toString('utf8')));
+    });
+  });
+});
+
+const cleanupProject = (projectName) => {
+  const projectDirectory = getProjectDirectory(projectName);
+  fse.removeSync(projectDirectory);
+};
+
+const initProject = async (projectName) => {
+  const projectDirectory = getProjectDirectory(projectName);
+  if (fs.existsSync(projectDirectory)) {
```

Comment:
Minor, but to reduce some duplicate code, I would just add this if-check to the `cleanupProject` function and then call that function here.